VBA/Excel-VBA/articles/excel-performance-and-limit-improvements.md

```diff
@@ -1,247 +1,165 @@
 
-### Copying Conditional Formats
-In Excel 2013 copy/pasting cells containing conditional formats could be slow.
-This has been significantly improved in Excel 2016 build 16.0.8229.0
+*My test on copy/pasting 22,000 rows filtered from 44,000 rows showed a dramatic improvement:*
+- *For a table, the time went from 39 seconds in Excel 2013 to 2 seconds in Excel 2016.*
+- *For a range, the time went from 30 seconds in Excel 2013 to virtually instantaneous in Excel 2016.*
 
-*My test on copying 44000 cells with a total of 386000 conditional format rules showed a substantial improvement:*
-- Excel 2013: 68 seconds
-- Excel 2016: 7 seconds
+### Copying conditional formats
+In Excel 2013, copy/pasting cells containing conditional formats could be slow. This has been significantly improved in Excel 2016 Build 16.0.8229.0.
 
-### Adding and deleting Worksheets
-My test on Excel 2016 build 16.0.8431.2058 shows a 15-20% speed improvement compared to Excel 2013 when adding and deleting large numbers of worksheets.
+*My test on copying 44,000 cells with a total of 386,000 conditional format rules showed a substantial improvement:*
+- *Excel 2013: 68 seconds*
+- *Excel 2016: 7 seconds*
 
+### Adding and deleting worksheets
+My test on Excel 2016 Build 16.0.8431.2058 shows a 15-20% speed improvement compared to Excel 2013 when adding and deleting large numbers of worksheets.
 
-### New Functions
+### New functions
 
-Excel 2016 build 16.0.7920.1000 introduced sevearl very useful new worksheet functions:
+Excel 2016 Build 16.0.7920.1000 introduced several new and very useful worksheet functions:
 
-**MAXIFS** and **MINIFS** extend the **COUNTIFS/SUMIFS** family of functions. These functions have good performance characteristics and should be used to replace equivalent array formulas.
-**SWITCH** and **IFS** provide ways of simplifying complex IF statements.
-**TEXTJOIN** and **CONCAT**  let you easily combine text strings from ranges of cells.
+- **MAXIFS** and **MINIFS** extend the **COUNTIFS/SUMIFS** family of functions. These functions have good performance characteristics and should be used to replace equivalent array formulas.
+- **SWITCH** and **IFS** provide ways of simplifying complex IF statements.
+- **TEXTJOIN** and **CONCAT**  let you easily combine text strings from ranges of cells.
 
-### Integrated Power Pivot and Power Query (Get & Transform)
-In Excel 2016 Power Pivot and Power Query (Get and Transform) are fully integrated to Excel rather than being Addins, resulting in improved performance and control.
+### Integrated PowerPivot and Power Query (Get & Transform)
+In Excel 2016, PowerPivot and Power Query (Get and Transform) are fully integrated to Excel rather than being add-ins, resulting in improved performance and control.
 
-### Other Updates to Excel 2016 for Windows
+### Other updates to Excel 2016 for Windows
 You can find more details of all the other month-by-month improvements that have been made to Excel 2016 at
```

Comment:
Revise as "For more details about the month-by-month improvements to Excel 2016, see... "